### PR TITLE
also mark dynflow containers as auto-started

### PR DESCRIPTION
**Pull request by evgeni**
Fixes: 371727b767dd7b012566138069ea08e54b00906b

damn template services

### CODE REVIEW

File: roles/foreman/tasks/main.yaml [hunk 1]
```diff
@@ -87,6 +87,20 @@
...
+
+- name: Create Dynflow Container instances
+  ansible.builtin.file:
+    state: link
+    src: "/etc/containers/systemd/dynflow-sidekiq@.container"
+    dest: "/etc/containers/systemd/dynflow-sidekiq@{{ item }}.container"
+  loop:
+    - orchestrator
+    - worker
+    - worker-hosts-queue
```
Comment (ehelms) [inline roles/foreman/tasks/main.yaml:103]: Tests for each of these? 

Does the container startup command derive what persona to be based on the name of the service?

File: roles/foreman/tasks/main.yaml [hunk 2]
```diff
@@ -87,6 +87,20 @@
       DYNFLOW_REDIS_URL: "redis://localhost:6379/6"
       REDIS_PROVIDER: "DYNFLOW_REDIS_URL"
     command: "/usr/libexec/foreman/sidekiq-selinux -e production -r /usr/share/foreman/extras/dynflow-sidekiq.rb -C /etc/foreman/dynflow/%i.yml"
```
Comment (ehelms) [inline roles/foreman/tasks/main.yaml:89]: I wonder if the `sidekiq-selinux` start up command will be needed inside a container.

File: roles/foreman/tasks/main.yaml (hunk 1 above)
Comment (evgeni) [reply roles/foreman/tasks/main.yaml:103]: Test that `/etc/containers/systemd/dynflow-sidekiq@{{ item }}.container` exists? Can add.

File: roles/foreman/tasks/main.yaml (hunk 2 above)
Comment (evgeni) [reply roles/foreman/tasks/main.yaml:89]: Maybe. Maybe not. Probably not.

**Pull request merged**